libs/features/mas/web-components/src/variants/product.js

```diff
@@ -82,5 +83,9 @@ export class Product extends VariantLayout {
         min-height: var(--consonant-merch-card-product-callout-content-height);
         display: block;
     }
+      
+    :host([variant='product']) ::slotted([slot='heading-xs']) {
+      max-width: var(--consonant-merch-card-heading-xs-max-width, 100%);
```

Comment:
is this still needed?

Reply:
Yes, this limits the title max-width for title.   "adjustTitleWidth" sets the calue of "--consonant-merch-card-heading-xs-max-width" and we need to apply the css here to set it into effect